src/logic/operation.test.js

```diff
@@ -0,0 +1,20 @@
+import operate from './operation';
+
+describe('checking if operations work properly', () => {
+  test('division', () => {
+    const result = operate('10', '2', '÷');
+    expect(result).toBe('5');
+  });
+  test('addition', () => {
+    const result = operate('6', '2', '+');
+    expect(result).toBe('8');
+  });
+  test('subtraction', () => {
+    const result = operate('10', '5', '-');
+    expect(result).toBe('5');
+  });
+  test('multiplication', () => {
+    const result = operate('2', '2', 'x');
+    expect(result).toBe('4');
+  });
+});
```

Comment:
- [ ] Please implement the following tests too, to cover all the features added in the project:

 - It should throw 0 errors if trying to make a zero division.
 - It should return 20. as next if the button name is.
 - It should give the percentage of the first operand if the button name is %.